guides/15_Background_Filter_Video_Processor.md

```diff
@@ -135,19 +160,35 @@ interface BackgroundBlurOptions {
 * blurStrength: The amount of blur that will be applied to a video stream. 
 * filterCPUUtilization: The threshold CPU utilization to trigger skipping background filter updates which will reduce the amount of CPU used by background filtering. For example, If the reporting period is set to 1000 ms and 500 ms was dedicated to processing the background filter, then the CPU utilization for that reporting period is 50%. If `filterCPUUtilization` is set to 50 it will cause a `filterCPUUtilizationHigh` event to be fired from the `BackgroundBlurVideoFrameProcessorObserver`.
 
+**BackgroundReplacementOptions**
+
+```typescript
+interface BackgroundReplacementOptions {
+  logger?: Logger;
+  reportingPeriodMillis?: number;
+  filterCPUUtilization?: number;
+  imageBlob?: Blob;
+}
+```
+
+The logger, reportingPeriodMillis, and filterCPUUtilization work in the exact same way as with `BackgroundBlurOptions`. 
+
+* imageBlob: This is a blob that contains an image that will be used in the background. If an image blob is not provided a solid color will be displayed. The Blob can be created in many different ways (e.g. https fetch, file upload, etc.). If you are using fetch to load the image blob we recommend using HTTPS for a security reasons.
```

Comment:
> If an image blob is not provided a solid color
Do we not know the color (black, green, etc)?

Reply:
Yes. I will add that to the documentation.